src/main/kotlin/demo/dto/application/ApplicationPayloads.kt

```diff
@@ -0,0 +1,54 @@
+package demo.dto.application
+
+import demo.models.offers.Application
+import demo.models.offers.ApplicationStatus
+import demo.models.offers.RecruitmentStage
+import demo.models.people.JobSeeker
+import demo.models.people.User
+import java.sql.Date
+import java.time.LocalDate
+import java.util.*
+
+data class ApplicationLoggedInPayload(val userId: Int) {
```

Comment:
Wydaje mi się, że to rozróżnienie jest niepotrzebne, bo oni i tak aplikują z wysłania formularza, więc wszystkie te informacje będziemy raczej mieli